session/txnInfo/txnInfo.go

```diff
@@ -0,0 +1,81 @@
+// Copyright 2021 PingCAP, Inc.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package txnInfo
```

Comment:
This file and the folder's names' format doesn't look consistent with others. Consider renaming the file to `session/txninfo/txn_info.go`, and the package name to `txninfo`.